response_operations_ui/templates/reporting-unit.html

```diff
@@ -78,8 +78,10 @@ <h4 class="neptune">Collection exercises</h4>
               </td>
               <td name="tbl-ce-status" class="table--cell">
                 {{ ce.responseStatus }}
-                {% if ce.statuses|length > 0 and (ce.responseStatus == 'Not started' or ce.responseStatus == 'In progress')  %}
-                &nbsp;<a href="{{ url_for('case_bp.get_response_statuses', ru_ref=ru.sampleUnitRef, survey=survey.shortName, period=ce.exerciseRef) }}">Change</a>
+                {% if ce.responseStatus in ['Not started', 'In progress']  %}
```

Comment:
Should this logic be taken out of the template?

Reply:
I think in this instance its ok here i consulted Owen about this he said its fine. Plus to get the logic into the reporting units build up would add alot there to an already convoluted process.